src/sentry/dynamic_sampling/rules/base.py

```diff
@@ -17,7 +17,7 @@
 from sentry.models import Organization, Project
 
 # These rules types will always be added to the generated rules, irrespectively of the base sample rate.
-ALWAYS_ALLOWED_RULE_TYPES = {RuleType.BOOST_LOW_VOLUME_PROJECTS_RULE}
+ALWAYS_ALLOWED_RULE_TYPES = {RuleType.BOOST_LOW_VOLUME_PROJECTS_RULE, RuleType.CUSTOM_RULE}
```

Comment:
I dislike the `custom rule` name tbh, I would be confused to read it in a while from now. What if we call it investigation rule?

Reply:
There will be other uses for this rules besides investigation (for example boosts for detected performance issues).
The name is not ideal ( especially since it may get confused with custom metrics).

I would welcome some suggestions, something to denote it is created from some sort of user interaction maybe.

Reply:
Ah ok, then it makes sense. I lacked context on this.